### PR TITLE
Fix repositories [SCI-1873]

### PR DESCRIPTION
**Pull request by okriuchykhin**
https://biosistemika.atlassian.net/browse/SCI-1873

### CODE REVIEW

File: app/datatables/repository_datatable.rb [hunk 1]
```diff
@@ -230,12 +232,12 @@ def filter_records(repo_rows)
 
   # Override default sort method if needed
   def sort_records(records)
-    if params[:order].present? && params[:order].length == 1
-      if sort_column(params[:order].values[0]) == ASSIGNED_SORT_COL
+    if order_params.present?
```
Comment (ZmagoD) [inline app/datatables/repository_datatable.rb]: as above it always evaluate true so why we need this? Before that was also a :banana: before params[:order].length == 1 
hmm...

File: app/datatables/repository_datatable.rb [hunk 2]
```diff
@@ -246,40 +248,26 @@ def sort_records(records)
...
-        dir = sort_direction(params[:order].values[0])
+        dir = sort_direction(order_params)
 
         records.joins(
-          "LEFT OUTER JOIN my_module_repository_rows ON
-          (repository_rows.id = my_module_repository_rows.repository_row_id
-          AND (my_module_repository_rows.my_module_id = #{@my_module.id} OR
-                            my_module_repository_rows.id IS NULL))"
-        ).order("my_module_repository_rows.id NULLS #{direction}")
+          "LEFT OUTER JOIN (SELECT repository_cells.repository_row_id,
```
Comment (ZmagoD) [inline app/datatables/repository_datatable.rb]: it was something wrong with the query?
Do this affect the datatable on task?

File: app/datatables/repository_datatable.rb [hunk 3]
```diff
@@ -335,10 +323,10 @@ def sort_assigned_records(records, direction)
       ids = unassigned + assigned
     end
 
-    order_by_index = ActiveRecord::Base.send(
+    order_by_index = ActiveRecord::Base.__send__(
```
Comment (ZmagoD) [inline app/datatables/repository_datatable.rb]: I don't think that ActiveRecord::Base overrides Object.send method. Do you have a good reason to use this?

File: app/datatables/repository_datatable.rb (hunk 2 above)
Comment (okriuchykhin) [reply app/datatables/repository_datatable.rb]: That was an old code, it was changed in master.

File: app/datatables/repository_datatable.rb (hunk 3 above)
Comment (okriuchykhin) [reply app/datatables/repository_datatable.rb]: Just linter was complaining about it

Comment (ZmagoD) [reply app/datatables/repository_datatable.rb]: fuck the linter this is not python :smile:

File: app/datatables/repository_datatable.rb (hunk 1 above)
Comment (okriuchykhin) [reply app/datatables/repository_datatable.rb]: Ok, removed that check

**Comment by ZmagoD** [comment]
Good job :ship:

**Pull request merged**